shell/renderer/electron_renderer_client.cc

```diff
@@ -125,16 +125,16 @@ void ElectronRendererClient::DidCreateScriptContext(
   render_frame->GetWebFrame()->GetDocumentLoader()->SetDefersLoading(
       blink::LoaderFreezeMode::kStrict);
 
+  v8::Isolate* const isolate = renderer_context->GetIsolate();
```

Comment:
Any reason to not call `v8::Isolate::GetCurrent()` here already ?

Reply:
Not sure I follow, isn't getting isolate from context is being deprecated ?

Reply:
the tldr is the plan was to refactor `RendererClientBase::DidCreateScriptContext()` to take a `v8::Isolate*` arg so that it wouldn't need to know where the isolate came from, but I agree that this step of the series feels incomplete as-is because it adds a new call to the deprecated API.

I've added ba655bc19621f8e41fdd179d1e289f2fc1b9c4ca to add the isolate pointer argument and to use `v8::Isolate::GetCurrent()` in the caller.